core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/cluster/Cluster.java

```diff
@@ -138,8 +140,8 @@
 
     @Convert(converter = SecretToString.class)
     @SecretValue
-    @Column(nullable = false)
-    private Secret extendedBlueprintText = Secret.EMPTY;
+    @Column(name = "extendedBlueprintText", nullable = false)
```

Comment:
can we rename the column ?

Reply:
I would like to do the migration at the last step to keep backward compatible as late as possible